sensing/pointcloud_preprocessor/src/outlier_filter/ring_outlier_filter_nodelet.cpp

```diff
@@ -362,40 +352,44 @@ void RingOutlierFilterComponent::setUpPointCloudFormat(
   formatted_points.is_bigendian = input->is_bigendian;
   formatted_points.is_dense = input->is_dense;
 
-  sensor_msgs::PointCloud2Modifier pcd_modifier(formatted_points);
-  pcd_modifier.setPointCloud2Fields(
-    num_fields, "x", 1, sensor_msgs::msg::PointField::FLOAT32, "y", 1,
-    sensor_msgs::msg::PointField::FLOAT32, "z", 1, sensor_msgs::msg::PointField::FLOAT32,
-    "intensity", 1, sensor_msgs::msg::PointField::FLOAT32);
+  // pcl::PCLPointCloud2 pcl_aux;
+  sensor_msgs::msg::PointCloud2 msg_aux;
+  // pcl_conversions::moveFromPCL(pcl_pc2, cloud);
+  pcl::toROSMsg(pcl::PointCloud<OutputPointType>(), msg_aux);
+  formatted_points.fields = msg_aux.fields;
+  // pcl::for_each_type<typename pcl::traits::fieldList<OutputPointType>::type>
+  // pcl::detail::FieldAdder<OutputPointType>(formatted_points.fields); pcl::for_each_type<typename
+  // pcl::traits::fieldList<OutputPointType>::type>
+  // (pcl::detail::FieldAdder<OutputPointType>(formatted_points.fields));
```

Comment:
Please either remove the commented-out lines or add an explanatory comment as to why they should stay here.

Reply:
Addressed in [6793245](https://github.com/autowarefoundation/autoware.universe/pull/6996/commits/6793245f20a97f874eb90e448ae51edaa38ddc82)